From ece5b3cd9e10844dbf41c733a14de4cb17cd3d85 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 28 Oct 2014 11:20:30 +0000
Subject: [PATCH] p2v: kickstart: Remove 'install' line.

'install' is the default operation, it doesn't have to be specified.

(cherry picked from commit 26952db56ffab1bc93c7f57c43e785f62d44854f)
---
 p2v/p2v.ks.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/p2v/p2v.ks.in b/p2v/p2v.ks.in
index 077db04..4173975 100644
--- a/p2v/p2v.ks.in
+++ b/p2v/p2v.ks.in
@@ -17,7 +17,6 @@
 
 # Generated by virt-p2v-make-kickstart.
 
-install
 lang en_US.UTF-8
 keyboard us
 timezone --utc GMT
-- 
1.8.3.1

